addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -564,6 +567,11 @@ def _l10n_es_edi_call_web_service_sign(self, invoices, info_list):
                 results[inv] = {'success': True}
                 inv.message_post(body=_("We saw that this invoice was sent correctly before, but we did not treat "
                                         "the response.  Make sure it is not because of a wrong configuration."))
+
+            elif respl.CodigoErrorRegistro == 1117 and not self.env.context.get('error_1117'):
+                self = self.with_context(error_1117=True)
```

Comment:
@yosa-odoo I am a bit afraid that the context would be used further on the line.  Why not just return self.with_context(error...)._post_invoice_edi(invoices) ?

Reply:
@jco-odoo I changed it :)

Reply:
(I was doing that at first, don't remember why I changed)